app/src/usb_hid.c

```diff
@@ -23,11 +23,78 @@ static K_SEM_DEFINE(hid_sem, 1, 1);
 
 static void in_ready_cb(const struct device *dev) { k_sem_give(&hid_sem); }
 
+#define HID_GET_REPORT_TYPE_MASK 0xff00
+#define HID_GET_REPORT_ID_MASK 0x00ff
+
+#define HID_REPORT_ID_KEYBOARD 1
+#define HID_REPORT_ID_CONSUMER 2
+
```

Comment:
These are defined now in `hid.h` now that #1993 is in. Please rebase and leverage those.